app/models/appdata.rb

```diff
@@ -34,7 +36,10 @@ def self.add_appdata data, xml
   # Get the appdata xml for a distribution
   def self.get_distribution dist="factory", flavour="oss"
     if dist == "factory"
-      appdata_url = "http://download.opensuse.org/tumbleweed/repo/#{flavour}/suse/setup/descr/appdata.xml.gz"
+      appdata_regex = /appdata-\S{32}.xml.gz/
+      tumbleweed_contents = open('http://download.opensuse.org/tumbleweed/repo/#{flavour}/content') {|f| f.read }
```

Comment:
Yep, sorry. See https://github.com/openSUSE/software-o-o/issues/73 for a fix I provided 3 hours ago. Thought you subscribed to the pull requests of this repository.